video/out/wayland_common.c

```diff
@@ -91,6 +92,11 @@ static int set_cursor_visibility(struct vo_wayland_state *wl, bool on)
     return VO_TRUE;
 }
 
+static void reset_cursor(struct vo_wayland_state *wl)
+{
+        set_cursor_visibility(wl, wl->cursor_visible);
```

Comment:
Why add a 1-line function that you're only going to call once?

Reply:
I totally agree. Like I said earlier, I only left it in since it was originally forked from t-8ch's PR. I'll happily remove it.